docs/wfc3tools/pstat.rst

```diff
@@ -4,25 +4,87 @@
 pstat
 *****
 
-Plot statistics for a specified image section  up  the  stack  of  a
+Plot statistics for a specified image section  up  the  stack  of
 an IR MultiAccum image.  Sections from any of the SCI,
 ERR, DQ,  image extensions can be  plotted.   A  choice
-of  mean,  median,  mode,  standard  deviation,  minimum and maximum
+of  mean,  median,  mode,  standard  deviation,  minimum, and maximum
 statistics is available.  The total number of samples is  determined
 from  the  primary  header  keyword NSAMP and all samples (excluding
 the zeroth-read) are plotted.  The SCI, ERR, DQ  statistics
 are  plotted as a function of sample time. The sample times  are  read
 from  the  SAMPTIME  keyword in the SCI header for each readout.
 
-SAMP and TIME aren't generally populated until the FLT image stage
-To plot the samptime vs sample, use wfc3tools.pstat and the "time" extension
+SAMP and TIME aren't generally populated until the FLT image stage.
+To plot the samptime vs sample, use wfc3tools.pstat and the "time" extension.
 
 
 .. figure:: ../_static/pstat_example.png
     :align: center
     :alt: pstat example plot
 
-    An example of the default plot from pstat
+
+.. Warning::
+    Note that the arrays are structured in SCI order, so the final exposure is the first element in the array
+
+
+Parameters
+==========
+
+* filename : str
+    Input   MultiAccum   image  name  with  optional  image  section  specification.  If no image section  is  specified,  the  entire image  is  used.   This  should  be  either a _raw or _ima file, containing all  the  data  from  multiple  readouts.   You  must specify  just  the  file name and image section, with no extname designation.
+
+* extname :  str, default="sci"
+    Extension name (EXTNAME keyword value) of data to plot. Allowed values are "sci", "err", and "dq".
+
+* units : str, default="counts"
+    Plot "sci" or  "err"  data  in  units  of  counts  or  countrate
+    ("rate").   Input data can be in either unit; conversion will be
+    performed automatically.  Ignored when  plotting  "dq",  "samp",
+    or "time" data. Allowed values are "counts" and "rate".
+
+* stat : str, default="midpt"
+    Type of statistic to compute. Allowed values are "mean", "midpt",
```

Comment:
Module docstring lists `median` instead of `midpt`, should check code and make consistent.